### DISCUSSION TITLE
FIDO2-based Passwordless Authentication

### DISCUSSION THREAD

**Discussion by jheiselman** [dani-garcia/vaultwarden#2960] [Ideas]
As announced on https://bitwarden.com/blog/passwordless-authentication-access-your-bitwarden-web-vault-without-a-password/ it would be awesome to see this supported in Vaultwarden as well.

**Comment by BlackDex** [comment]
It currently isn't even supported on there self hosted environment. So that would maybe not be that easy.

**Reply by jheiselman** [reply]
Totally understandable. Just wanting to get the discussion started in the event that it does show up in their self-hosted code.
